keyboards/alu84/kb.c

```diff
@@ -0,0 +1 @@
+#include "kb.h"
```

Comment:
These `kb` files should be named the same as the project - in this case, `alu84` :)

Reply:
will fix.